src/test/java/org/jbehavesupport/runner/story/steps/TestSteps.java

```diff
@@ -34,10 +30,45 @@
 public class TestSteps {
 
     @BeforeStory
-    public void before() {
+    public void beforeStory() {
         logger.info("Before story custom step");
     }
 
+    @BeforeStories
+    public void beforeStories() {
+        logger.info("Before Stories custom step");
+    }
+
+    @BeforeScenario
+    public void beforeScenario() {
+        logger.info("Before scenario custom step");
+    }
+
+    @BeforeScenario
+    public void beforeScenario2() {
+        logger.info("Second before scenario custom step");
+    }
+
+    @AfterScenario
+    public void afterScenario() {
+        logger.info("After scenario custom step");
+    }
+
+    @AfterScenario
+    public void afterScenario2() {
+        logger.info("Second after scenario custom step");
+    }
+
+    @AfterStory
+    public void afterStory() {
+        logger.info("After story custom step");
+    }
+
+    @AfterStories
+    public void afterStories() {
+        logger.info("After stories custom step");
+    }
+
     private static final Logger logger = LoggerFactory.getLogger(TestSteps.class);
```

Comment:
Again nitpick, but could you move this field Logger logger to somewhere along line 30? And yes I know its my own mistake to put it in here 😄